vhdbuilder/packer/vhd-image-builder.json

```diff
@@ -199,6 +199,16 @@
       "source": "parts/linux/cloud-init/artifacts/docker-monitor.timer",
       "destination": "/home/packer/docker-monitor.timer"
     },
+    {
```

Comment:
wonder if it's better to move containerd/docker monitor service files from vhd to cse so we can use provision-time templating to avoid duplication.

my thought was this bit of duplication is fine since docker will get deprecated as we introduce newer k8s versions

Reply:
good call. the duplicate should be fine, maybe we should remove the other one when we ensure dockerd or containerd.